src/cmd.ts

```diff
@@ -335,17 +338,9 @@ createCommandWithSharedOptions("daemon", "Start the cross-seed daemon")
 	)
 	.action(async (options) => {
 		try {
-			const runtimeConfig = processOptions(options);
-			setRuntimeConfig(runtimeConfig);
-			initializeLogger();
-			initializePushNotifier();
-			logger.info(`${PROGRAM_NAME} v${PROGRAM_VERSION}`);
-			logger.verbose({
-				label: Label.CONFIGDUMP,
-				message: inspect(runtimeConfig),
-			});
+			const errorCount = await validateAndSetRuntimeConfig(options);
 			await db.migrate.latest();
-			await doStartupValidation();
+			await doStartupValidation(errorCount);
```

Comment:
This, yes. I'm trying to parse the path's at the same time as the Zod errors. It will ultimately crash if any errors exist after parsing with Zod and checking paths, providing all the errors that exist at once, rather than incrementally through subsequent runs as it encounters them and they are fixed.

I did this, albeit crudely, to keep track of whether errors were encountered, mostly for the aesthetics/messaging at the end.

Reply:
After running into a few more potential problems during testing the error handling, I think with integrating the path formatting check into Zod and the hurdles with passing around counts and stuff, I'm just going to revert back to crashing on Zod validation failure.

The benefit of producing path errors in addition to Zod on subsequent runs seems like a very edge-case situation that I don't think is beneficial for us to account for with it requiring such a break from the existing (and the convention you mention) "architecture".

Will be pushing a commit shortly to address this and a few other things.

Reply:
Address in latest push.